.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.7
```

Comment:
What's the point of this file?  Doesn't the circle config yml lock us onto a specific version in Circle?

Reply:
Mostly copying the convention from api. I think if local devs need it that can add it and add it to gitignore. Still not sure if this file is strictly for things like rbenv.

Reply:
But yeah, the build seems to work file without it.